RELEASING.md

```diff
@@ -50,8 +56,8 @@ This will:
 
 ### Verify the Release
 
-You can validate the staged artefacts by having a test application (we'll call it `T`) that uses kroxylicious use the Maven artefacts by making
-temporary (local) changes to its POM.
+You can validate the staged artefacts by using a test application, `T`, use the Maven artefacts.   The [kroxylicious-wasm](https://github.com/andreaTP/kroxylicious-wasm) from the 
+[community-gallery](https://github.com/kroxylicious/kroxylicious-community-gallery) is a suitable choice.
 
 1. Find the staging repository in the [Nexus UI](https://s01.oss.sonatype.org/). It'll be named `iokroxylious-nn`.
 1. Add a [`<repositories>`](https://maven.apache.org/pom.html#Repositories) that references the staging repository public url to `T`'s POM.
```

Comment:
I have 
```
    <profile>
      <id>test-release</id>
      <repositories>
        <repository>
            <id>pending</id>
            <url>https://s01.oss.sonatype.org/content/repositories/staging/</url>
            <releases>
                <enabled>true</enabled>
            </releases>
            <snapshots>
                <enabled>false</enabled>
            </snapshots>
        </repository>
    </repositories>
    </profile>
```
in `${HOME}/.m2/settings.xml` so I would just need to bump the sample repo to using the RC version and run `mvn clean install -Ptest-release`. 

I think we could build that into a pom in the gallery repo and be just run that as a github action 😁 I think thats a bigger fix though.

I think the comment for this PR is that we can add `https://s01.oss.sonatype.org/content/repositories/staging` as the URL for the repository in the docs and stop mentioning `iokroxylious-nn`.

Reply:
Are you sure about that?   In my mental model nexus repos are entirely private.  They don't get amalgamated into a single public staging repo.  Indeed, nexus lets you have many staging repos active at once, so how would it know how to resolve conflicts?

Reply:
I think I tested it previously but I can stage a release to validate it and open a fresh pr if it works.

As I agree with your mental model